<ui version="4.0" >
 <class>UiImportExportPage</class>
 <widget class="QWizardPage" name="UiImportExportPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Import / Export</string>
  </property>
  <layout class="QGridLayout" name="glImportExport" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBoxImport" >
     <property name="title" >
      <string>Option</string>
     </property>
     <layout class="QGridLayout" name="glImport" >
      <item row="0" column="0" >
       <widget class="QRadioButton" name="rbtnImport" >
        <property name="text" >
         <string>Import D-Fend profiles</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QRadioButton" name="rbtnExport" >
        <property name="text" >
         <string>Export DBoxFE profiles to D-Fend profiles</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBoxDirectories" >
     <property name="title" >
      <string>Directories</string>
     </property>
     <layout class="QGridLayout" name="glDirectory" >
      <item row="0" column="0" >
       <layout class="QHBoxLayout" name="glDFend" >
        <item>
         <widget class="QLabel" name="labelDFend" >
          <property name="minimumSize" >
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>110</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>D-Fend (Rel.):</string>
          </property>
          <property name="buddy" >
           <cstring>lineEditDFendDirectory</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditDFendDirectory" />
        </item>
        <item>
         <widget class="QPushButton" name="btnDFend" >
          <property name="minimumSize" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" name="glDboxFe" >
        <item>
         <widget class="QLabel" name="labelDboxFe" >
          <property name="minimumSize" >
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>110</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>DBoxFE:</string>
          </property>
          <property name="buddy" >
           <cstring>lineEditDboxFeDirectory</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditDboxFeDirectory" />
        </item>
        <item>
         <widget class="QPushButton" name="btnDboxFe" >
          <property name="minimumSize" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QTreeWidget" name="treeWidgetImportExport" >
     <property name="showDropIndicator" stdset="0" >
      <bool>false</bool>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="itemsExpandable" >
      <bool>false</bool>
     </property>
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>Profile</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" name="hlMark" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnMarkAll" >
       <property name="minimumSize" >
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Mark all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUnmarkAll" >
       <property name="minimumSize" >
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Unmark All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>rbtnImport</tabstop>
  <tabstop>rbtnExport</tabstop>
  <tabstop>lineEditDFendDirectory</tabstop>
  <tabstop>btnDFend</tabstop>
  <tabstop>lineEditDboxFeDirectory</tabstop>
  <tabstop>btnDboxFe</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
